_common/KeymanHosts.php

```diff
@@ -82,5 +93,15 @@ function __construct() {
         $this->keymanweb_com = "{$site_protocol}keymanweb.com{$site_suffix}";
         $this->r_keymanweb_com = "https://r.keymanweb.com"; /// local dev domain is usually not available
       }
+
+      $this->blog_keyman_com_host = preg_replace('/^http(s)?:\/\/(.+)$/', '$2', $this->blog_keyman_com);
+      $this->s_keyman_com_host  = preg_replace('/^http(s)?:\/\/(.+)$/', '$2', $this->s_keyman_com);
+      $this->api_keyman_com_host = preg_replace('/^http(s)?:\/\/(.+)$/', '$2', $this->api_keyman_com);
```

Comment:
Are these to manipulate in-place URLs?  Basically, to take, say, "https://api.keyman.com/foo/bar" and point it to "https://api.keyman-staging.com/foo/bar" if in a 'staging' tier?

Reply:
These are basically hacky `"https://api.keyman.com"` to `"api.keyman.com"` conversions. I am actually not very happy with them but am going to leave fixing them for a rainy day.